common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -422,9 +422,16 @@ private ErrorStrings() {
     public final static String DEVICE_CODE_FLOW_EXPIRED_TOKEN_CODE = "expired_token";
     public final static String DEVICE_CODE_FLOW_EXPIRED_TOKEN_MESSAGE = "The token has expired, therefore authentication is no longer possible with this flow attempt. Re-run the Device Code Flow Protocol to try again.";
 
+    /**
+     * Device Code Flow only.
+     * The token was polled again after it was already received.
+     */
+    public final static String DEVICE_CODE_FLOW_INVALID_GRANT_CODE = "invalid_grant";
```

Comment:
We _should_ have a constant for this somewhere already, let me track it down

Reply:
Let's use this value here:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java#L1414

Reply:
will do!